### ISSUE TITLE
can't reproduce cifar10

### ISSUE THREAD

**Issue by twmht** [aicaffeinelife/Pytorch-STN#1]
Can't reproduce cifar10 result.

Here is mine

without stn: 78.3%
with stn: 76.7%

Any idea?

**Comment by aicaffeinelife** [comment]
What optimizer are you using?

**Comment by twmht** [comment]
sgd

**Comment by twmht** [comment]
Does this code can reproduce the result?

**Comment by aicaffeinelife** [comment]
yeah, this code uses `Adam` optimizer. Can you try using that? If not I'll take a closer look again

Thanks

**Comment by twmht** [comment]
@aicaffeinelife 

I use adam and still can't repoduce the result

without stn: 78.4
with stn: 76.1

**Comment by tkone2018** [comment]
hello,when the epoch=82,the value is 77.3,after epoch=82,the value is very low,i don't know how to interpret it,can you help me ?ths

**Comment by tkone2018** [comment]
can you teach me how to use the STN with vgg16 or resnet50?ths

**Comment by aicaffeinelife** [comment]
@tkone2018 open a separate issue for this one. Using STN in any CNN is easy, but it's not relevant to this thread.

**Comment by aicaffeinelife** [comment]
@twmht I just cloned the repo and re-ran experiments on the hardware mentioned in `README.md`. I got `78.19%` accuracy. What hardware were you using?

**Comment by tkone2018** [comment]
@aicaffeinelife ok

**Comment by twmht** [comment]
@aicaffeinelife 

GTX-1080

**Comment by aicaffeinelife** [comment]
@twmht  that is strange. Can you share your `params.json` here?
